pkg/config/config.go

```diff
@@ -14,16 +14,16 @@ var (
 		field.WithRequired(true),
 		field.WithIsSecret(true),
 	)
-	EnterpriseTokenField = field.StringField(
-		"enterprise-token",
-		field.WithDisplayName("Enterprise Token"),
-		field.WithDescription("The Slack user oauth token used to connect to the Slack Enterprise Grid Admin API"),
+	BusinessPlusTokenField = field.StringField(
+		"business-plus-token",
```

Comment:
Should we just keep the old name? I'm sure Slack will rebrand this product in the future, and we don't want to break backward compatibility every time they do that.